project-templates/import-records/Makefile

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
Yeah I see little value in Grunt personally, I prefer Make, especially for projects that are mixed with Javascript and other tools.